flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/queryConfig.scala

```diff
@@ -92,6 +100,23 @@ class StreamQueryConfig private[table] extends QueryConfig {
     this
   }
 
+  /**
+    * Specifies a deferred computation time for deferred computation.
+    */
+  def withDeferredComputationTime(deferredComputationTime: Time): StreamQueryConfig = {
+
+    if (deferredComputationTime.toMilliseconds < 1) {
```

Comment:
I think the default value is 0, If user want using this method, I think tell user the useful config value is very important. What do you think ?